File: WMS.ScheduleTasks/SnapShot.cs
using System;
using System.Data;
using Made4Net.DataAccess;
using Made4Net.Shared;
using WMS.Logic;

namespace WMS.SchedulerTasks
{
	/// <summary>
	/// Summary description for SnapShot.
	/// </summary>
	public class SnapShot
	{
		public SnapShot() 
		{

		}

        public void Execute()
        {
            Made4Net.Shared.QMsgSender oQMsg = new Made4Net.Shared.QMsgSender();
            oQMsg.Add("EVENT", 36.ToString());
            oQMsg.Add("ACTIVITYDATE", Made4Net.Shared.Util.DateTimeToWMSString(DateTime.Now));
            oQMsg.Send("EventManager", "36", "", System.Messaging.MessagePriority.Normal);
        }

        public void ExecuteConsigneeSnapShot(string pConsignee)
        {
            Made4Net.Shared.QMsgSender oQMsg = new Made4Net.Shared.QMsgSender();
            oQMsg.Add("EVENT", 36.ToString());
            oQMsg.Add("CONSIGNEE", pConsignee);
            oQMsg.Add("ACTIVITYDATE", Made4Net.Shared.Util.DateTimeToWMSString(DateTime.Now));
            oQMsg.Send("EventManager", "36", "", System.Messaging.MessagePriority.Normal);
        }
	}
}

File: WMS.Logic/Processes/SnapShot.vb

<CLSCompliant(False)> Public Class SnapShot
    Inherits Made4Net.Shared.JobScheduling.ScheduledJobBase

    Public Sub New()

    End Sub

    Public Overrides Sub Execute()
        'Dim aq As New EventManagerQ
        'Dim action As String = WMS.Lib.Actions.Audit.SNAPSHOT
        'aq.Add("ACTION", action)
        'aq.Add("ACTIVITYDATE", Made4Net.Shared.Util.DateTimeToWMSString(DateTime.Now))
        'aq.Send(action)
        Try
            Dim em As New EventManagerQ
            Dim EventType As Int32 = WMS.Logic.WMSEvents.EventType.SnapShot
            em.Add("EVENT", EventType)
            em.Add("ACTIVITYDATE", Made4Net.Shared.Util.DateTimeToWMSString(DateTime.Now))
            em.Send(WMSEvents.EventDescription(EventType))
        Catch ex As Exception
        End Try
    End Sub
End Class
